notifications_utils/markdown.py

```diff
@@ -67,6 +68,8 @@ def block_quote(self, text):
     def header(self, text, level, raw=None):
         if level == 1:
             return super().header(text, 2)
+        elif level == 2:
+            return super().header(text, 3)
```

Comment:
I don’t think we want to tackle headings in letters yet because there’s some additional complication there (the [story](https://trello.com/c/OLMQ3ZSy/389-add-subheadings-to-available-markdown) does say ‘for emails only’)
```suggestion
```

Reply:
I'll remove it for letters 👍  One of the open questions was also `do we want to do this for letters?` so I did it in case. But easily removed 👍 )